mapadroid/cache/__init__.py

```diff
@@ -0,0 +1,21 @@
+from mapadroid.cache.noopcache import NoopCache
+from mapadroid.utils.logging import get_logger, LoggerEnums
+
+logger = get_logger(LoggerEnums.system)
+
+
+def get_cache(args):
+    cache = NoopCache()
+    if args.enable_cache:
+        try:
+            import redis
+            cache = redis.Redis(host=args.cache_host, port=args.cache_port, db=args.cache_database)
```

Comment:
We should try to keep this connection open to reduce the overhead of the connection process. Maybe if we initialize it we should store it as a singleton to save some of the processing time of re-opening and pining.